nidmviewerfsl/callSlicer.py

```diff
@@ -0,0 +1,182 @@
+#=======================================================================================================
+#
+# The following functions are designed to resize SPM nifti maps to align with an FSL template and create
```

Comment:
@TomMaullin: I have a question here. Could you explain shortly why we need to align with the FSL template? Is it impossible to use the FSL `slicer` command but remain in SPM standard space?

Reply:
[this is a short report of discussions we had by email with @TomMaullin]
 - Here we are not doing registration but only reslicing/resampling of the excursion set onto the FSL template.

We discussed:
 - Using the corresponding template (e.g. SPM template) when a NIDM-Results pack is displayed (cf. the `inWorldCoordinateSystem` variable) which means no reslicing should be needed. @TomMaullin will create an issue to remind this for another PR.
 - Using nibabel from now on for operations on NIfTI images as this is easier in Python. For display  we will use FSL functions as much as we can so that we can more easily reproduce the FSL output.

@TomMaullin: Are you fine with this report of our discussions? Feel free to add more info.